.github/CODEOWNERS

```
@@ -74,3 +74,10 @@
 # ./lib/internal/modules/* @nodejs/modules
 # ./lib/internal/bootstrap/loaders.js @nodejs/modules
 # ./src/module_wrap* @nodejs/modules @nodejs/vm
+
+# N-API
+
+# ./src/node_api* @nodejs/n-api
+# ./src/js_native_api* @nodejs/n-api
+# ./doc/guides/adding-new-napi-api.md @nodejs/n-api
+# ./doc/api/n-api.md @nodejs/n-api
```

Uncomment out to enable, and github does not like the leading `.` at the start of the paths.

```suggestion
/src/node_api* @nodejs/n-api
/src/js_native_api* @nodejs/n-api
/doc/guides/adding-new-napi-api.md @nodejs/n-api
/doc/api/n-api.md @nodejs/n-api
```